mythril/mythx/__init__.py

```diff
@@ -79,8 +79,10 @@ def analyze(contracts: List[SolidityContract], analysis_mode: str = "quick") ->
             issue = Issue(
                 contract=contract.name,
                 function_name=None,
-                address=issue.locations[0].source_map.components[0].offset,
-                swc_id=issue.swc_id[4:],  # remove 'SWC-' prefix
+                address=issue.locations[0].source_map.components[0].offset
```

Comment:
can you also put in some text which says the user is using trail mode and to visit mythx.io for pro.

Reply:
It can be done in the place where we use default address and password.

Reply:
:+1: